update.sh

```diff
@@ -8,44 +8,52 @@
 
 set -e
 
+# Check whether there is a newer version of this script
+cd /usr/share/rhino/rhino-update
+if git pull; then
+    chmod +x /usr/share/rhino/rhino-update/update.sh
+    exec /usr/bin/rhino-update
+    exit $?
+fi
+
 # Check to see whether the "configuration update", released in 2022.04.19 has been applied.
-if [[ ! -f "$HOME/.rhino/updates/configuration" ]]; then
-  mkdir -p ~/.rhino/{config,updates}
-  echo "alias rhino-config='mkdir ~/.rhino/config/config-script && git clone https://github.com/rollingrhinoremix/rhino-config ~/.rhino/config/config-script/ && python3 ~/.rhino/config/config-script/config.py && rm -rf ~/.rhino/config/config-script'" >> ~/.bashrc
-  : > "$HOME/.rhino/updates/configuration"
+if [[ ! -f "/usr/share/rhino/updates/configuration" ]]; then
```

Comment:
Why not use a variable for `/usr/share/rhino`? This avoids repetition in case you need to change directories.

Reply:
That isn't a bad idea but that would require more work for this PR and I think this could be incorporated in a different PR

Reply:
As for `#!/usr/bin/env bash` I don't think it is necessary as this script is only made to run on a specific Linux distro.

Reply:
To be fair it's better design and universal. It's a much better approach despite not being useful in this specific use case. And most importantly, it doesn't increase maintenance cost.

Reply:
I agree with Tesk's approach personally.